src/ckeditor/ckeditor.component.ts

```diff
@@ -121,15 +121,25 @@ export class CKEditorComponent implements AfterViewInit, OnDestroy, ControlValue
 	 */
 	@Output() ready = new EventEmitter<CKEditor4.EventInfo>();
 
+	/**
+	 * Fires when the editor data is loaded, e.g. after calling setData()
+	 * https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-setData
+	 * editor's method. It corresponds with the `editor#dataReady`
+	 * https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-dataReady event.
+	 */
+	@Output() dataReady = new EventEmitter<CKEditor4.EventInfo>();
+
 	/**
 	 * Fires when the content of the editor has changed. It corresponds with the `editor#change`
 	 * https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-change
 	 * event. For performance reasons this event may be called even when data didn't really changed.
+	 * Please note that this event will only be fired when `undo` plugin is loaded. If you need to
+	 * listen for editor changes (e.g. for two-way data binding), use `dataChange` event instead.
 	 */
 	@Output() change = new EventEmitter<CKEditor4.EventInfo>();
 
 	/**
-	 * Fires when the content of the editor has changed. In contrast to `change` - only emits when
+	 * Fires when the content of the editor has changed. In contrast to `change` and `selectionCheck` - only emits when
```

Comment:
```suggestion
	 * Fires when the content of the editor has changed. In contrast to `change` - only emits when
```

Since `selectionCheck` is no longer exposed it shouldn't be mentioned here.